pubsublite/manager.go

```diff
@@ -240,3 +272,56 @@ func (m *Manager) DeleteSubscription(ctx context.Context, subscription string) e
 	logger.Info("deleted pubsublite subscription")
 	return nil
 }
+
+func (m *Manager) gatherMetrics(ctx context.Context, o metric.Observer) error {
+	it := m.monitoringClient.ListTimeSeries(ctx, &monitoringpb.ListTimeSeriesRequest{
+		Name:   fmt.Sprintf("projects/%s", m.cfg.Project),
+		Filter: "metric.type = \"pubsublite.googleapis.com/subscription/backlog_message_count\"",
+		Interval: &monitoringpb.TimeInterval{
+			StartTime: &timestamppb.Timestamp{Seconds: time.Now().Add(-5 * time.Minute).Unix()},
```

Comment:
[for reviewer] I tried starttime == endtime and it didn't like it. Not even with start time = now -1m, so I just set start time = now -5m and it just works.